ios/FluentUI.Demo/FluentUI.Demo/Resources/Assets.xcassets/TableViewCell/excelIcon.imageset/Contents.json

```diff
@@ -2,14 +2,11 @@
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "excelIcon.pdf"
+      "filename" : "excelIcon.png"
```

Comment:
assuming it is the same size, no impact on the tableviewcell demo?

Reply:
no impact! not sure why it was png to begin with but I can include TVC screenshots as well. size remains the same.